openid.js

```diff
@@ -75,7 +77,7 @@ module.exports = function (RED) {
     const state = req.query.state.split(':')
     const node_id = state[0]
     const credentials = RED.nodes.getCredentials(node_id)
-    if (!credentials || !credentials.client_id || !credentials.client_secret) {
+    if (!credentials || !credentials.client_id || !credentials.client_secret || !credentials.scopes) {
```

Comment:
Scope is not part of the credentials validation. It's OK as soon you have a client_id and secret.
```suggestion
    if (!credentials || !credentials.client_id || !credentials.client_secret) {
```